utils/constants.jsx

```diff
@@ -1429,6 +1429,15 @@ export const Constants = {
 
 export const AcceptedProfileImageTypes = ['image/jpeg', 'image/png', 'image/bmp'];
 
+export const searchHintOptions = [{searchTerm: 'From:', message: {id: 'search_list_option.from', defaultMessage: 'Messages from a user'}},
+    {searchTerm: 'In:', message: {id: 'search_list_option.in', defaultMessage: 'Messages in a channel'}},
```

Comment:
These still need to be wrapped with `t` to have the i18n tools pick them up like
```suggestion
    {searchTerm: 'In:', message: {id: t('search_list_option.in'), defaultMessage: 'Messages in a channel'}},
```